LEAF_Request_Portal/Login.php

```diff
@@ -181,7 +181,7 @@ public function loginUser()
             $authType = '/auth_cookie/?r=';
             $nonBrowserAuth = '/auth_cookie/?r=';
           }
-
+          $_SESSION['userID'] = 'VACOLayJ';//TODO MAKE STATELESS LOGIN WORK CORRECTLY
```

Comment:
Can this be the tester user, dont think you should include this specific username

Reply:
done